File: problemSolving/programmers/level_1_java/두_개_뽑아서_더하기.java
import java.util.ArrayList;
import java.util.Arrays;

class Solution {
    public int[] solution(int[] numbers) {
        
        ArrayList<Integer> list = new ArrayList<Integer>();
        
        for (int i = 0; i < numbers.length - 1; i++) {
            for (int j = i + 1; j < numbers.length; j++) {
                int num = numbers[i] + numbers[j];
                if (list.indexOf(num) < 0){
                    list.add(num);
                };
            };
        };
        int[] answer = new int[list.size()];
        for (int i = 0; i < list.size(); i++) {
            answer[i] = list.get(i);
        }
        Arrays.sort(answer);
            
        return answer;
    }
}

File: problemSolving/programmers/level_1_js/두개뽑아서더하기.js
function solution(numbers) {
  var answer = [];
  numbers.sort();
  for (let i = 0; i < numbers.length; i++) {
    for (let j = i + 1; j < numbers.length; j++) {
      if (
        answer.find((item) => item === numbers[i] + numbers[j]) === undefined
      ) {
        answer.push(numbers[i] + numbers[j]);
      }
    }
  }
  answer.sort((a, b) => a - b);
  return answer;
}
